content/react/concepts/hooks/terms/useState/useState.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: 'useState()'
+Description: 'Returns the current state of the component and its setter. State must be changed through the setter only and not be mutated directly as it may cause unexpected behavior.'
+Subjects:
+  - 'Web Development'
+Tags:
+  - 'React'
+  - 'useState'
+CatalogContent:
+  - 'react-101'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `useState()` returns the current state of the component and its setter. State must be changed through the setter only and not be mutated directly as it may cause unexpected behavior.
+
+## Syntax
+
+The `useState()` hook accepts an initial value as its parameter. If no value is passed, it will be `undefined`.
+
+The return value is an array whose first and second elements are the current state and state setter respectively. Those variables can be obtained through array destructuring.
```

Comment:
```suggestion
```

For the "Syntax" sections, let's begin the text below the \```pseudo code block. 😄

Reply:
Looks like this was resolved without this being deleted. 😮

Reply:
oops must've missed that